src/components/Popup/PopupContent.vue

```diff
@@ -76,6 +79,10 @@ export default defineComponent({
     to: {
       type: String,
       default: 'body'
+    },
+    disableBackdropClose: {
```

Comment:
he's not here to validate, but I believe Joe might have wanted clicking the backdrop _not_ to close the popup. I could be confusing that with him _not_ wanting a "close" button. @marichka-offen / @zhen0 / @ThatGalNatalie does that ring a bell?

Reply:
Yes that's right. I remember Joe mentioning that

Reply:
Hm... this is an important question; I believe expected behavior for modals is that they can be closed when an overlay is clicked unless doing so could disrupt important business in the modal itself. 

I added this because right now the burden is on the developer to provide a close button of some sort tied to the model. I opted for overlay closing as the _default_ behavior for this reason; persistent keyboard traps should be an opt-in scenario for the developer since they can be a really frustrating experiences for users. Given that this provides the option for either one, are we happy with that?

Reply:
Added a comment in Figma to track 👍🏻